packages/web-twig/src/Resources/components/ButtonLink/stories/ButtonLinkColorsSizes.twig

```diff
@@ -0,0 +1,18 @@
+<section class="docs-Section">
+  <h2 class="docs-Heading">Default</h2>
+
+  <Grid cols="1" desktop="3">
+    {% for size in sizes %}
+    <div>
+      <h3 class="docs-Heading">Size {{ size }}</h3>
+      {% for color in colors %}
+      <div class="{% if color is same as('inverted') %}docs-Box {% endif %}my-600">
+        <div>
+          <ButtonLink color={color} size={size} href="#">Button {{ color }}</ButtonLink>
```

Comment:
I would love to see a square variant here as well (and in the disabled state)

Reply:
Ok, I have added them.